skipper.go

```diff
@@ -66,6 +53,10 @@ type Options struct {
 	// Skip TLS certificate check for etcd connections.
 	EtcdInsecure bool
 
+	// Kubernetes API URL for ingress. By default, Kubernetes ingress is
+	// disabled.
+	KubeUrl string
```

Comment:
I would propose using the long form "kubernetes" everywhere (we don't need to save any ASCII chars, do we?)